cookbook/profiler/profiling_data.rst

```diff
@@ -0,0 +1,60 @@
+.. index::
+   single: Profiling; Profiling data
+
+How to Access Profiling Data Programmatically
+=============================================
+
+Most of the times, the profiler information is accessed and analyzed using its
+web-based visualizer. However, you can also retrieve profiling information
+programmatically thanks to the methods provided by the ``profiler`` service.
+
+When the response object is available, use the
+:method:`Symfony\\Component\\HttpKernel\\Profiler\\Profiler::loadProfileFromResponse`
+method to access to its associated profile::
+
+    $profile = $container->get('profiler')->loadProfileFromResponse($response);
```

Comment:
we should not advocate the usage of `$contaier->get()` imo. It also doesn't tell important information, such as: Where should I place this?